pkg/cri/server/instrumented_service.go

```diff
@@ -232,13 +235,16 @@ func (in *instrumentedService) StopPodSandbox(ctx context.Context, r *runtime.St
 	if err := in.checkInitialized(); err != nil {
 		return nil, err
 	}
+	ctx, span := tracing.StartSpan(ctx, tracing.Name(criSpanPrefix, "StopPodSandbox"))
```

Comment:
again .. not sure we need to start span in instrumented, and Stop()and stop()  these id's won't always match

Reply:
We create a new span in the instrumented service and then we reference the same span in Stop() and stop(). Since each attribute in a span has to be unique, otel only keeps the last updated value for a attribute. In our case `container.id` and `sandbox.id` cannot have multiple values on the same span.

Anyways I will make the change to use the full id wherever I have added the attribute